setup.py

```diff
@@ -30,12 +30,17 @@
 import versioneer
 
 from setuptools import setup, find_packages
+import yaml
```

Comment:
Is `yaml` really available at `setup`-time?

Reply:
well, that's kind of a problem. The other solutions could include:
- having a separate "requirements.txt" that always matches the environment.yml
- parsing the yaml file "by hand" whitout using an external lib
- finding some way to "include" the requirements.txt in the .yml file

All in all, users are supposed to build the conda env in which the setup.py is run beforehand, and that env already has PyYaml installed, so it's not a problem.